### PR TITLE
Feat alternative hubble report

### PR DESCRIPTION
**Pull request by florian-hoenicke**
What do you think about this way of reporting to hubble. From my understanding, it is easier to understand and to maintain.

### CODE REVIEW

**Comment by florian-hoenicke** [comment]
image
Client deployment is still working

**Review by IyadhKhalfallah — APPROVED** [review]
LGTM 🚀

File: now/app/base/app.py [hunk 1]
```diff
@@ -118,7 +118,11 @@ def get_gateway_stub(self, user_input, testing=False) -> Dict:
             'uses_with': {'user_input_dict': user_input.to_safe_dict()},
             'env': {'JINA_LOG_LEVEL': 'DEBUG'},
             'jcloud': {
-                'resources': {'instance': 'C5', 'capacity': 'spot'},
+                'resources': {
+                    'instance': 'C5',
+                    'capacity': 'spot',
+                    'storage': {'kind': 'ebs', 'size': '10G'},
```
Comment (joschkabraun) [inline now/app/base/app.py:124]: does the gateway need `10G`?

**Review by azayz — APPROVED** [review]

File: now/app/base/app.py (hunk 1 above)
Comment (florian-hoenicke) [reply now/app/base/app.py]: no, thanks - I will change

**Pull request merged**